src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -121,4 +121,40 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
         }
         return tagSet;
     }
+
+    /**
+     * Returns the String evenTitle passed to it.
```

Comment:
```suggestion
     * Returns the String eventTitle passed to it.
```
Minor spelling error

Reply:
done